docs-source/content/samples/simple/elastic-stack/soa-domain/weblogic-monitoring-exporter-setup.md

```diff
@@ -0,0 +1,305 @@
+
+```
+$ ./get1.1.0.sh config-admin.yaml
+ % Total % Received % Xferd Average Speed Time Time Time Current
+ Dload Upload Total Spent Left Speed
+100 607 0 607 0 0 915 0 --:--:-- --:--:-- --:--:-- 915
+100 2016k 100 2016k 0 0 839k 0 0:00:02 0:00:02 --:--:-- 1696k
+created /tmp/ci-H1SNbxKo1b
+/tmp/ci-H1SNbxKo1b ~/weblogic-monitor/tmp
+in temp dir
+ adding: config.yml (deflated 63%)
+~/weblogic-monitor/tmp
+$ ls
+config-admin.yaml get1.1.0.sh wls-exporter.war
+```
+
+Similarly, you must generate the deployment package for the Managed Servers and cluster with a different configuration file.
+
+#### Deploy the WebLogic Monitoring Exporter
+
+Follow these steps to deploy the package in the WebLogic Server instances.
+
+1. Deploy the WebLogic Monitoring Exporter (`wls-exporter.war`) in the Administration Server and Managed Servers separately using the Oracle Enterprise Manager.
+
+1. Select the servers to deploy.
+
+1. Set the application name. The application name has to be different if it is deployed separately in the Administration Server and Managed Servers.
+
+1. Set the context-root to `wls-exporter` for both the deployments.
+
+1. Select **Install and start application**.
+
+1. Then deploy the WebLogic Monitoring Exporter application (`wls-exporter.war`).
+
+1. Activate the changes to start the application. If the application is started and the port is exposed,
+then you can access the WebLogic Monitoring Exporter console using this URL, `http://<server:port>/wls-exporter`.
+
+#### Prometheus Operator configuration
+
+You must configure Prometheus to collect the metrics from the WebLogic Monitoring Exporter. The Prometheus Operator identifies the targets using service discovery.
+To get the WebLogic Monitoring Exporter end point discovered as a target, you must create a service monitor pointing to the service.
+
+See the following sample service monitor deployment YAML configuration file.  
+
+`ServiceMonitor` for wls-exporter:
+```
+apiVersion: v1
+kind: Secret
+metadata:
+  name: basic-auth
+  namespace: monitoring
+data:
+  password: V2VsY29tZTE= # Welcome1 i.e.'WebLogic password'
+  user: d2VibG9naWM= # weblogic  i.e. 'WebLogic username'
+type: Opaque
+---
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: wls-exporter-soainfra
+  namespace: monitoring
+  labels:
+    k8s-app: wls-exporter
+spec:
+  namespaceSelector:
+    matchNames:
+    - soans
+  selector:
+    matchLabels:
+      weblogic.domainName: soainfra
+  endpoints:
+  - basicAuth:
+      password:
+        name: basic-auth
+        key: password
+      username:
+        name: basic-auth
+        key: user
+    port: default
+    relabelings:
+      - action: labelmap
+        regex: __meta_kubernetes_service_label_(.+)
+    interval: 10s
+    honorLabels: true
+    path: /wls-exporter/metrics
+```
+
+The exporting of metrics from `wls-exporter` requires basic authorization so a secret, created with the user name and password, are base64 encoded.
```

Comment:
basicAuth, Secret, ServiceMonitor are Kubernetes key words. Please see line 213. So the ine 229 needs be rephrased as below:

The exporting of metrics from `wls-exporter` requires basicAuth. So a Kubernetes Secret is created with the user name and password that are base64 encoded. This Secret will be used in the ServiceMonitor deployment.

Reply:
I understand. I wrote it as follows: "The exporting of metrics from `wls-exporter` requires `basicAuth` so a Kubernetes `Secret` is created with the user name and password that are base64 encoded. This `Secret` will be used in the `ServiceMonitor` deployment."